src/tests.zig

```diff
@@ -347,17 +366,17 @@ test "deserializes a string" {
 }
 
 const Pastry = struct {
-    name: []const u8,
+    name: []u8,
```

Comment:
deserialization only worked because it was returning a direct pointer to the serialized payload, which is `[]const u8` itself.